types/v56/types.go

```diff
@@ -2132,6 +2132,7 @@ type QueryResultRecordsType struct {
 	VMRecord                        []*QueryResultVMRecordType                        `xml:"VMRecord"`                        // A record representing a VM result.
 	AdminVMRecord                   []*QueryResultVMRecordType                        `xml:"AdminVMRecord"`                   // A record representing a Admin VM result.
 	VAppRecord                      []*QueryResultVAppRecordType                      `xml:"VAppRecord"`                      // A record representing a VApp result.
+	OrgVdcRecord                    []*QueryResultOrgVdcRecordType                    `xml:"OrgVdcRecord"`                    // A record representing VDC
```

Comment:
What are we going to do with this field? No functions are using it.
Also, do we need an `AdminOrgVdcRecord` field?

Reply:
1. I added a method for usage this type and a test.
2. I don't think so, I didn`t find  AdminOrgVdcRecord in the schema and docs https://code.vmware.com/apis/722/vmware-cloud-director